tests/frontend/mirror.py

```diff
@@ -895,9 +895,29 @@ def test_source_mirror_plugin(cli, tmpdir):
 @pytest.mark.parametrize("unaliased_sources", [True, False])
 @pytest.mark.parametrize("disallow_subproject_uris", [True, False])
 @pytest.mark.parametrize("fetch_source", ["aliases", "mirrors"])
-@pytest.mark.parametrize("alias_override", [["foo"], ["foo", "bar"], "global", "invalid"])
+@pytest.mark.parametrize(
```

Comment:
Can we have a comment around here explaining why this level of parameterized testing is needed, and what defects we are specifically protecting against ?

Reply:
This is mainly to be able to exclude some elements from the test matrix.

Initially I wanted something like
```
@pytest.mark.parametrize("alias_override", [["foo"], ["foo", "bar"], "global"])
@pytest.mark.parametrize("alias_mapping", ["identity", "project-prefix", "invalid"])
```

but "global,project-prefix" isn't implemented yet (I'd like to have it in a future release) and "global,invalid" doesn't make sense.

As for the reason I'm doing this, the bug in question happens when having a non-identity mapping along with source mirror plugin. When I started, I thought it could be unrelated to source plugins, so wanted to test it.

If you have a better option, feel free to comment. Or I can add the above to a comment?

Reply:
Maybe calling `pytest.skip()` for these two cases (with a comment) would be reasonable.

Reply:
Added a comment.